### PR TITLE
added Facebook Profile Picture

### PR DESCRIPTION
**Pull request by DEN379**
added Facebook Profile Picture url

### CODE REVIEW

File: Services/FacebookOAuth/FacebookProfileManager.cs [hunk 1]
```diff
@@ -62,7 +62,7 @@ private async Task<UserProfile> GetProfileFromAccessTokenAsync(FacebookTokensMod
             string fields = ReformScopeToFields.Transform(tokensModel.Scope);
             
             string url = facebookProperties.GetProfileLink + tokensModel.AccessToken +
-                    "&fields=first_name,last_name,name,gender,location" + fields;
+                    "&fields=first_name,last_name,name,gender,location,picture" + fields;
```
Comment (Vadimur) [inline Services/FacebookOAuth/FacebookProfileManager.cs:65]: Is it only for test?

File: Providers/FacebookUserProfileInfoProvider.cs [hunk 2]
```diff
@@ -7,17 +7,26 @@
...
     {
+        /// <summary>
+        /// Mapps UserProfile fields
+        /// </summary>
+        /// <param name="facebookInfo">FacebookInfoModel</param>
+        /// <returns>UserProfile</returns>
         public static UserProfile Provider(FacebookInfoModel facebookInfo)
         {
-            return new UserProfile()
+                return new UserProfile()
```
Comment (Vadimur) [inline Providers/FacebookUserProfileInfoProvider.cs]: Fix tabulation

File: Providers/FacebookUserProfileInfoProvider.cs [hunk 3]
```diff
@@ -7,17 +7,26 @@
...
+    /// Provider from FacebookInfoModel to UserProfile
+    /// </summary>
     public static class FacebookUserProfileInfoProvider
     {
+        /// <summary>
+        /// Mapps UserProfile fields
+        /// </summary>
+        /// <param name="facebookInfo">FacebookInfoModel</param>
+        /// <returns>UserProfile</returns>
         public static UserProfile Provider(FacebookInfoModel facebookInfo)
```
Comment (Vadimur) [inline Providers/FacebookUserProfileInfoProvider.cs:20]: Are you sure that facebookInfo always not null?

File: Services/FacebookOAuth/FacebookProfileManager.cs (hunk 1 above)
Comment (DEN379) [reply Services/FacebookOAuth/FacebookProfileManager.cs:65]: It's default fields for getting main information about user from Facebook

File: Providers/FacebookUserProfileInfoProvider.cs (hunk 2 above)
Comment (DEN379) [reply Providers/FacebookUserProfileInfoProvider.cs]: Ok, i'll fix this

File: Providers/FacebookUserProfileInfoProvider.cs (hunk 3 above)
Comment (DEN379) [reply Providers/FacebookUserProfileInfoProvider.cs:20]: Not sure, need to add validation

**Review by Vadimur — APPROVED** [review]

**Pull request merged**